spec/quotas/ianaconsiderations.mdown

```diff
@@ -12,4 +12,4 @@ Intended use: common
 
 Change Controller: IETF
 
-Security and privacy considerations: this document, section 4.
\ No newline at end of file
+Security and privacy considerations: this document, section 3.
```

Comment:
maybe a less brittle way to do this is referencing an anchor to the relevant section and let the markdown-to-xml2rfc processor resolve it.